src/components/AddPeopleAndRoles/ListPeopleAndRoles.vue

```diff
@@ -139,22 +141,22 @@ import { Getter } from 'vuex-class'
   }
 })
 export default class ListPeopleAndRoles extends Mixins(CommonMixin, EntityFilterMixin) {
-  // Store Properties
   @Prop({ default: () => [] })
-  private personList: Array<OrgPersonIF>
+  private readonly personList: Array<OrgPersonIF>
```

Comment:
Props are readonly. Maybe you're seen the Jest warning "can't set a readonly property". In theory this should warn us in our IDEs if we try to assign something to a prop.